projects/wizard_of_wikipedia/wizard_transformer_ranker/wizard_transformer_ranker.py

```diff
@@ -0,0 +1,123 @@
+# Copyright (c) 2017-present, Facebook, Inc.
+# All rights reserved.
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree. An additional grant
+# of patent rights can be found in the PATENTS file in the same directory.
+
+from parlai.agents.transformer.transformer import TransformerRankerAgent
+from parlai.core.torch_agent import TorchAgent
+
+import numpy as np
+import torch
+
+
+class WizardTransformerRankerAgent(TransformerRankerAgent):
+    @classmethod
+    def add_cmdline_args(cls, argparser):
+        """Add command-line arguments specifically for this agent."""
+        super(WizardTransformerRankerAgent, cls).add_cmdline_args(argparser)
```

Comment:
nit: `super().add_cmdline_args` works here

Reply:
does it??!

Reply:
oops, no you're right nevermind